README.md

```diff
@@ -160,6 +160,13 @@ set -ag status-right "#{E:@catppuccin_status_session}"
 set -ag status-right "#{E:@catppuccin_status_uptime}"
 set -agF status-right "#{E:@catppuccin_status_battery}"
 
+# automatically set CWD as window name
+set -g @catppuccin_window_default_text "#W"
```

Comment:
This option no longer exists

Reply:
Sorry, took it from https://github.com/catppuccin/tmux/issues/53#issuecomment-1675211611.